maintainer/conda/MDAnalysis/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+MDA_USE_OPENMP=FALSE pip install package/
```

Comment:
I am curious: @hainm , do you not have this issue (can't ship OpenMP-enabled binaries) with pytraj or do you not use OpenMP code?